CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.2.0 (July 28, 2022)
+
+IMPROVEMENTS:
+
+* Update `project_key` attribute validation to expand characters limit from 3-10 to 3-25. PR: [#48](https://github.com/jfrog/terraform-provider-project/pull/48)
+* Fix module vulnerability from [Dependabot](https://github.com/jfrog/terraform-provider-project/security/dependabot/2)
+
```

Comment:
I don't see the field update anywhere, except in the CHANGELOG.md. 
See this - https://github.com/jfrog/terraform-provider-project/blob/44ede3ec4a5c780abfb7ffe63bb0cda381643ceb/pkg/project/resource_project.go#L60